src/localization/validationStrings.js

```diff
@@ -24,6 +24,7 @@ export const validationStrings = new LocalizedStrings({
     repeatPassword: {
       required: 'Enter the repeat password',
     },
+    is_required: 'is required',
```

Comment:
I am reluctant to pick this particular scab again... but here goes...
Shouldn't this property be `isRequired`?

Reply:
eba5dc6

Don't be reluctant please! I didn't notice the strings in this file were using camelCase :(

I've made an issue here to try and refactor soon for consistency https://github.com/openmsupply/mobile/issues/1695